api/graphql/schema.graphql

```diff
@@ -634,6 +635,9 @@ type ApplicantFilter {
   expectedClassifications: [Classification]
     @belongsToMany(relation: "classifications")
   skills: [Skill] @belongsToMany
+  classifications: [Classification]
+    @belongsToMany(relation: "classifications")
```

Comment:
type ApplicantFilter now has two classifications fields connected to it, this is kind of confusing, are they addressing the same thing because if so there should only be one

https://github.com/GCTC-NTGC/gc-digital-talent/blob/9990f8b224afba81f60e8008dde9ed86d80b92dc/api/graphql/schema.graphql#L635-L639

Reply:
Agree its very confusing. Both are 2 different fields and both are live in production. However, some of the places we inject `classifications` into `expectedClassifications` and rename it back into `classifications` while storing into db. 
I started cleaning up and left with the bundle of tangled threads. So thought to stay away from that in this PR.

Reply:
that is a bit unfortunate
I supposed please include a comment then that quickly explains what each classification field does? to make it easier to understand what is going on in the schema 

"some of the places we inject classifications into expectedClassifications and rename it back into classifications"
😭

Reply:
https://github.com/GCTC-NTGC/gc-digital-talent/issues/5133

Reply:
Expected Classification was originally used meant to capture this classification. But it has been changed as above now
![Screenshot 2023-04-04 at 10 53 32 AM](https://user-images.githubusercontent.com/89482910/229832330-c95ff45f-e31f-4b45-86f0-912e808bd26a.png)

Reply:
Yeah, it's a mess. :cry: Whatever you end up doing, please add some good comments to the schema file to make it clear what's going on.

Reply:
Was wondering whether to spin out an issue for removing `expectedClassification` to remove opportunity for this confusion to surface again, and found this related issues: https://github.com/GCTC-NTGC/gc-digital-talent/issues/5347

Reply:
thats right Pat! I attempted to rename only related pages in this PR. The whole cleanup should be in separate PR as in some places we are using `expectedClassification`  name to store `GenericJobTitles` .. thats making it more worse.. 😩```
expectedClassification: getExpectedClassifications(
          expectedGenericJobTitles,
          intl,
        ),
```